outputs.tf

```diff
@@ -1,11 +0,0 @@
-output "private_endpoints" {
-  description = "A map of private endpoints. The map key is the supplied input to var.private_endpoints. The map value is the entire azurerm_private_endpoint resource."
-  value       = azurerm_private_endpoint.this
-}
-
-# Module owners should include the full resource via a 'resource' output
-# https://azure.github.io/Azure-Verified-Modules/specs/terraform/#id-tffr2---category-outputs---additional-terraform-outputs
-output "resource" {
-  description = "This is the full output for the resource."
-  value       = azurerm_resource_group.TODO # TODO: Replace this dummy resource azurerm_resource_group.TODO with your module resource
-}
```

Comment:
Maybe we should keep this output according to the (docs)[https://azure.github.io/Azure-Verified-Modules/specs/terraform/#id-tffr2---category-outputs---additional-terraform-outputs] and output `azurerm_application_security_group.this`

Reply:
Hi @CiucurDaniel, I was only removing it passes the pr and pre-commit checks.

I was planning on leaving that to you so you can get hands on?

Let me know if you'd prefer me to add it :)